group_vars/dev.yml

```diff
@@ -191,6 +193,45 @@ users:
     update_password: on_create
 
 
+
+  - username: rsmith
+    name: "Ryan Smith"
+    groups:
+      - spg
+    ssh_key:
+      - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDY+5PsNWI+aw4argp/2951uduln0gs+iYSOiiB4HCh5ScHd3ZLf2uhSvPqGQlc+zQcU/tx4b4YyAKNHXWX1eW+cf9FV1hakQrEFJ5fxagnSD6fY2z8Dwgxt0lLPQ0JFtl44OK890oZEvVKzcuKrYGV82SjkSqFuvTKxgWSinKHTvxDXqzFS4ZhUBYHSYtm75W0s7WL591YnstddvKfAUAYpMOfe3f0gLA0t/4hwg40W3/Ii8OafvYDVmVKb7m1Oyg8+safpCE4uNSewrCjtsf/8qgJ26QW4FVyB2srtIU0HHuZnyQQEtzCk3poZKV4JEfWPBbQfJcqdowuDtMDDDrD ryan.smith@W10180665"
```

Comment:
This key seems very short...

Please ensure all users generate a new key for this system as key reuse is the same as password reuse and should be discouraged. 

Keys should be at least 4096-bit at a minimum and should be 16348 unless there is a good reason otherwise.